docs/TROUBLESHOOTING.md

```diff
@@ -99,6 +99,18 @@ This section includes how to debug a failed restore. For more specific issues re
  
 <hr style="height:1px;border:none;color:#333;">
 
+<h3 align="center">Restore Errors and Warnings<a id="creds"></a></h3>
+
+Restore errors and warnings are shown in the `veleror describe` output in three groups:
+1. Velero: Warnings or errors that relate to the overall restore workflow but not to a particular resource will be shown here.
+2. Cluster: Warnings or errors that relate to particular cluster-scoped resources will be shown here
+3. Namespaces: Warnings or errors that relate to particular namespaced resources will be shown here, grouped by namespace
+
+One or more errors in any of the above categories will cause a Restore to be `PartiallyFailed` rather than `Completed`. Warnings will not cause a change to the completion status.
+
+Note that if there are "Velero" errors (but no resource-specific errors), it is possible that the restore completed without any actual problems with restoring workloads, but careful validation of post-restore applications is advisable.
+For example, if there are PodVolumeRestore and/or Node Agent-related errors, check the status of PodVolumeRestores and DataDownloads -- if none of these are failed or still running, then volume data may have been fully restored.
```

Comment:
I would like to expand line 112 with sudo instructions of which resources to manually investigate in addition to PodVolumeRestore, perhaps this could be just a list of objects.  wdyt?

Reply:
@weshayutin Which resources did you have in mind? For restores, we have the restore itself and the PVR/DD -- do you mean actual restored resources or velero CRs?

Reply:
Well sir... correct me if I'm wrong, but here's what I'm thinking.  
Either:
  * velero restore describe --details has a resource by resource and velero's state.   I wonder if anything in (failed) can be checked if it exists and status could be (present - failed restore)
  * OR
  * We script out a helper script that cycles through anything in failed and give the user the status or yaml output.  Not sure.   
  
Obviously not to be done in this PR, however for this PR:
   * adding a note to check velero restore describe --details
   * The user should have enough experience to then to further investigate.

The rest of what you have IMHO is very helpful.

Reply:
Ahh ok. So instead of expanding what I'm saying in 111/112 (which is specifically addressing non-item "velero" errors, I should add another note around the other error types (cluster and namespace-scoped errors, which relate to restored items. So if there are item errors, then look at the yaml/status of those items -- I'll add a note around this as well.